cellxgene_schema_cli/scripts/gene_processing.py

```diff
@@ -169,11 +170,50 @@ def process_gene_info(gene_info: dict) -> None:
     """
     print("download", gene_info["description"])
     temp_file_path, _ = urllib.request.urlretrieve(gene_info["url"])
-    print("process", gene_info["description"])
-    gene_info["processor"](temp_file_path, gene_info["new_file"])
+    previous_ref_filepath = os.path.join(env.ONTOLOGY_DIR, f"previous_{gene_info['description']}.csv.gz")
+    try:
+        # temporarily backup previous processed csv
+        os.rename(gene_info["new_file"], previous_ref_filepath)
+        print("process", gene_info["description"])
```

Comment:
is this poor naming going to come back to haunt us?

Reply:
What poor naming? I'm not opposed to renaming anything here for clarity